kview/src/main/resources/dev/ikm/komet/kview/mvvm/view/descriptionname/description-name.fxml

```diff
@@ -21,7 +21,7 @@
     <top>
         <HBox alignment="TOP_CENTER" styleClass="edit-concept-label-container" BorderPane.alignment="TOP_CENTER">
             <children>
-                <Label fx:id="editDescriptionTitleLabel" alignment="TOP_CENTER" text="Edit Description: Other Name" textAlignment="CENTER" />
+                <Label fx:id="editDescriptionTitleLabel" alignment="TOP_CENTER" styleClass="edit-concept-title" text="Edit Description: Other Name" textAlignment="CENTER" />
```

Comment:
I don't think styling is in scope.  I didn't repeat the style from concept to pattern because the figmas aren't exactly the same.

Reply:
I'm seeing now that a comment on the ticket had styling matching... I think this is a discussion where we need Rohit and Carl to weigh in because our typically try to match the Figma